src/main/java/spoon/refactoring/CtRenameLocalVariableRefactoring.java

```diff
@@ -84,9 +84,9 @@ public void accept(CtReference t) {
 
 		@Override
 		public ScanningMode enter(CtElement element) {
-			if (ignoredParent != null && element instanceof CtElement) {
-				CtElement ele = (CtElement) element;
-				if (ele.hasParent(ignoredParent)) {
+			CtElement ele = element;
```

Comment:
I don't understand the purpose of local variable ele here: IMHO you can just remove this variable and the first if.

Reply:
@surli 
Done.